ISSUE_TEMPLATE/stable.md

```diff
@@ -111,6 +111,21 @@ curl -H 'Accept: application/json' 'https://updates.coreos.fedoraproject.org/v1/
 
 NOTE: In the future, most of these steps will be automated.
 
+## Update the release notes
+
+NOTE: You can do this simultaneously for all releases that are happening at the same time.
+
+- For each [issue that is pending a stable release in the tracker](https://github.com/coreos/fedora-coreos-tracker/issues?q=is%3Aissue+label%3Astatus%2Fpending-stable-release):
+  - [ ] Add an entry to the release notes file: `release-notes/stable.yml`
+  - [ ] Add a message to the issue saying that the fix as been released:
+    ```
+    The fix for this went into the stable stream release X.Y.Z. Please try out the new release and report issues.
```

Comment:
```suggestion
    The fix for this went into the `stable` stream release `X.Y.Z`.
```

I haven't been putting the "Please try out the new release" part because hopefully by the time it reaches stable people following the issue have confirmed it was fixed for them in `testing`.

I'm not firm on that, though.